specification/securityinsights/resource-manager/readme.md

```diff
@@ -66,6 +66,18 @@ directive:
 
 ---
 
+### Tag: package-2021-03-preview-only
+
+These settings apply only when `--tag=package-2021-03-preview-only` is specified on the command line.
+
+```yaml $(tag) == 'package-2021-03-preview-only'
+input-file:
+- Microsoft.SecurityInsights/preview/2021-03-01-preview/Settings.json
+- Microsoft.SecurityInsights/preview/2021-03-01-preview/operations.json
+```
+
+---
+
```

Comment:
Move it to the above the 2019 tag. 
The order should be 
composite tag for new to old
version tags new to old